src/dataflow-types/src/client/controller/compute.rs

```diff
@@ -430,6 +432,9 @@ where
                 finishing,
                 map_filter_project,
                 target_replica,
+                // Obtain an `OpenTelemetryContext` from the thread-local tracing
+                // tree to forward it on to the compute-side
```

Comment:
I know it's not so clear at the moment, but this is already "compute-side"; it's just going from the controller to the worker.

```suggestion
                // tree to forward it on to the compute worker.
```